emberjs-app/.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
```

Comment:
The contents of this file should be moved to `.github/workflows/`. I recommend separating each job into its own file and then running it from the existing ci.yml.

I spoke with Andrew and I'll make these changes myself and add to this PR.